ts/features/itwallet/issuance/screens/ItwIssuanceCredentialFailureScreen.tsx

```diff
@@ -44,9 +52,29 @@ type ContentViewProps = { failure: CredentialIssuanceFailure };
  */
 const ContentView = ({ failure }: ContentViewProps) => {
   const machineRef = ItwCredentialIssuanceMachineContext.useActorRef();
+  const storedCredential =
+    ItwCredentialIssuanceMachineContext.useSelector(selectCredential);
 
-  const closeIssuance = () => machineRef.send({ type: "close" });
-  const retryIssuance = () => machineRef.send({ type: "retry" });
+  const closeIssuance = () => {
+    machineRef.send({ type: "close" });
+    trackWalletCreationFailed({
+      reason: failure.type,
+      cta_category: "custom_2",
+      cta_id: I18n.t(
```

Comment:
We should get this info from the failures map below

Reply:
Do you mean only the reason info or all the info passed to the trackWalletCreationFailed function?

Reply:
See this: https://github.com/pagopa/io-app/blob/3d421d7f7b14d97fb035fd515fbeec99b49644e6/ts/features/itwallet/issuance/screens/ItwIssuanceCredentialFailureScreen.tsx#L83
We are mapping the CTA content from the failure. We should make sure to pass the correct string to the `trackWalletCreationFailed`.